src/components/Process/Create/index.tsx

```diff
@@ -441,16 +452,21 @@ export const ProcessCreate = () => {
   const { isOpen: isResetFormModalOpen, onOpen: onResetFormModalOpen, onClose: onResetFormModalClose } = useDisclosure()
   const sidebarMargin = useBreakpointValue({ base: 0, md: '350px' })
   const blocker = useBlocker(methods.formState.isDirty)
-  const { client, account } = useClient()
+  const { client, account, fetchAccount } = useClient()
   const { setStepDoneAsync } = useOrganizationSetup()
   const { isSubmitting, isSubmitSuccessful } = methods.formState
   const description = methods.watch('description')
 
-  // Trigger confirmation modal when form is dirty and user tries to navigate away
   useEffect(() => {
-    const isBlocked = blocker.state === 'blocked'
+    const fetch = async () => {
+      await fetchAccount()
+    }
+    fetch()
+  }, [])
```

Comment:
Add a comment explaining why is this effect here (I'd add it with an `IMPORTANT` to start with, since it's a fact).

Also, in these cases, simply use self-invoking anonymous function: 

```ts
  // IMPORTANT: We need to ensure the electionIndex is updated before creating an election.
  useEffect(() => {
    ;(async () => {
      await fetchAccount()
    })()
  }, [])
```

After adding the comment myself though, I realize that we could have issues if that person opens a new tab and creates a process in that new tab... we actually might need to do this fetch always BEFORE submitting (previously we were doing this after submitting, but I now realize that's a mistake).